src/components/ProductListContainer/index.js

```diff
@@ -0,0 +1,101 @@
+        groupCapacity: 10,
+        subscriptorsNumber: 8,
+        occupancyPercentage: 80
+    },
+    {
+        id: 2,
+        productPhotoUrl: 'https://placehold.co/310x210?text=Imagen',
+        listPrice: '$250.000,00',
+        discountedPrice: '$200.000,00',
+        discountPercentage: 20,
+        expirationDate: new Date().toISOString(),
+        description: 'Esta es un tarjeta de producto de prueba.',
+        colors: ['azul','blanco','negro'],
+        groupCapacity: 10,
+        subscriptorsNumber: 8,
+        occupancyPercentage: 80
+    },
+    {
+        id: 3,
+        productPhotoUrl: 'https://placehold.co/310x210?text=Imagen',
+        listPrice: '$250.000,00',
+        discountedPrice: '$200.000,00',
+        discountPercentage: 20,
+        expirationDate: new Date().toISOString(),
+        description: 'Esta es un tarjeta de producto de prueba.',
+        colors: ['azul','blanco','negro'],
+        groupCapacity: 10,
+        subscriptorsNumber: 8,
+        occupancyPercentage: 80
+    },
+    {
+        id: 4,
+        productPhotoUrl: 'https://placehold.co/310x210?text=Imagen',
+        listPrice: '$250.000,00',
+        discountedPrice: '$200.000,00',
+        discountPercentage: 20,
+        expirationDate: new Date().toISOString(),
+        description: 'Esta es un tarjeta de producto de prueba.',
+        colors: ['azul','blanco','negro'],
+        groupCapacity: 10,
+        subscriptorsNumber: 8,
+        occupancyPercentage: 80
+    }
+]
+
+const ProductListContainer = ({ productIds, featuredProductId }) => {
+    return (
+        <div>
+            {fakeProductsList.map(({
+                id,
+                productPhotoUrl,
+                listPrice,
+                discountedPrice,
+                discountPercentage,
+                expirationDate,
+                description,
+                colors,
+                groupCapacity,
+                subscriptorsNumber,
+                occupancyPercentage
+            }) => {
+                return <ProductCard
+                            key={id}
+                            productPhotoUrl={productPhotoUrl}
+                            listPrice={listPrice}
+                            discountedPrice={discountedPrice}
+                            discountPercentage={discountPercentage}
+                            expirationDate={expirationDate}
+                            description={description}
+                            colors={colors.join(', ')}
+                            groupCapacity={groupCapacity}
+                            subscriptorsNumber={subscriptorsNumber}
+                            occupancyPercentage={occupancyPercentage}
+                            extended={id === featuredProductId}
+                        />
+            })}
+        </div>
+    );
+};
+
+ProductListContainer.propTypes = {
+    productIds: PropTypes.arrayOf(PropTypes.number),
+    featuredProductId: PropTypes.number
```

Comment:
These should be `PropTypes.string`. IDs are typically implemented as UUID, like so `e2a95e8e-64b2-4375-9242-2a4c830b62ab`.

Fakedata can have an ID of `'1'`, otherwise the code gets too polluted with real UUIDs, but it's datatype should be something that can represent a real ID/UUID.